Src/FluentAssertions/Equivalency/Matching/MappedPathMatchingRule.cs

```diff
@@ -12,6 +12,11 @@ internal class MappedPathMatchingRule : IMemberMatchingRule
     private readonly MemberPath expectationPath;
     private readonly MemberPath subjectPath;
 
+    /// <summary>
+    /// Initializes a new instance of the <see cref="MappedPathMatchingRule"/> class.
+    /// </summary>
+    /// <exception cref="ArgumentNullException"><paramref name="expectationMemberPath"/> is <c>null</c> or empty.</exception>
```

Comment:
Using `ArgumentNullException` for "empty" is considered a bad practice. `ArgumentException` should be used instead.